otherlibs/win32unix/system.c

```diff
@@ -17,27 +17,32 @@
 #include <caml/memory.h>
 #include <caml/alloc.h>
 #include <caml/signals.h>
+#include <caml/u8tou16.h>
 #include "unixsupport.h"
 #include <process.h>
 #include <stdio.h>
 
-CAMLprim value win_system(cmd)
-     value cmd;
+CAMLprim value win_system(value cmd)
 {
   int ret;
   value st;
-  char *buf;
-  intnat len;
-
+  CRT_CHAR *buf;
+#ifndef HAS_WINAPI_UTF16
+  size_t len;
+#endif
   caml_unix_check_path(cmd, "system");
+#ifdef HAS_WINAPI_UTF16
+  buf = Crt_str_val(cmd);
+#else /* HAS_WINAPI_UTF16 */
   len = caml_string_length (cmd);
   buf = caml_stat_alloc (len + 1);
   memmove (buf, String_val (cmd), len + 1);
+#endif /* HAS_WINAPI_UTF16 */
```

Comment:
This seems a bad rebase as well, compare https://github.com/ocaml/ocaml/pull/153/files#diff-6456858f78e5820f603ddc0a5c3998fbR35.